HerPortal/Controllers/SignOutController.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Mvc;
+
+namespace HerPortal.Controllers;
+
+[Route("sign-out")]
+public class SignOutController : Controller
+{
+    public IActionResult Index()
+    {
+        HttpContext.SignOutAsync();
+
+        return RedirectToAction("Index", "StaticPages");
```

Comment:
It looks like you need to be logged in to view the index page so this will take you straight back to the login screen?

Reply:
Yep, this is intentional behaviour -- if there are no pages publically accessible, then on logout I would expect to be shown the login screen straight away.

If you'd prefer, there can instead be a single public landing page with only a "sign in" button?

Reply:
That makes sense. However I'd expect the default page to be the page with the list of files - I don't think there are plans for a static index page.
Fine if you're planning to change this later on.